src/cli/subcommands/snapshot_cmd.rs

```diff
@@ -46,6 +46,9 @@ pub enum SnapshotCommands {
         /// Tipset to start the export from, default is the chain head
         #[arg(short, long)]
         tipset: Option<i64>,
+        /// How many state-roots to include. Lower limit is 900 for `calibnet` and `mainnet`.
```

Comment:
Do you think there should be some sort of warning if the user provides a depth <900? With the current changes, this is the behavior on my machine if I do so, and I wouldn't know why it's "failing" without reading the code comment:
<img width="495" alt="Screen Shot 2023-08-07 at 10 45 19 AM" src="https://github.com/ChainSafe/forest/assets/11778588/ecd38d93-e48a-4852-9888-aa9eb0ab2755">